From eeceae5505103b98f146dd8984b5a76dc67ba941 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Fri, 28 Apr 2017 11:06:48 +0800
Subject: [PATCH] drm: bridge: dw-hdmi: remove the function is_rockchip

The function is_rockchip isn't used any more now that phy reset
operation is performed based on detected phy type.

Change-Id: I58e7a222bc1e1578f0d5d2fcd884b17171fb9601
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 include/drm/bridge/dw_hdmi.h | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/include/drm/bridge/dw_hdmi.h b/include/drm/bridge/dw_hdmi.h
index dda259da124a..f5447b6a4d34 100644
--- a/include/drm/bridge/dw_hdmi.h
+++ b/include/drm/bridge/dw_hdmi.h
@@ -92,18 +92,6 @@ struct dw_hdmi_plat_data {
 			     unsigned long mpixelclock);
 };
 
-static inline bool is_rockchip(enum dw_hdmi_devtype dev_type)
-{
-	switch (dev_type) {
-	case RK3288_HDMI:
-	case RK3368_HDMI:
-	case RK3399_HDMI:
-		return true;
-	default:
-		return false;
-	}
-}
-
 void dw_hdmi_unbind(struct device *dev, struct device *master, void *data);
 int dw_hdmi_bind(struct device *dev, struct device *master,
 		 void *data, struct drm_encoder *encoder,
-- 
2.35.3

